doc/release.md

```diff
@@ -98,13 +98,13 @@ git checkout -b release/v${new_version}
 ```
 and update the version number in `pyproject.toml` and `flake.nix`:
 ```
-sed -i -e "s|version = .*$|version = \"${new_version}${rc}\"|g" pyproject.toml flake.nix
+sed -i -e "s|version = .*$|version = \"${new_version}\"|g" pyproject.toml flake.nix
```

Comment:
This will break the `flake.nix` (because there are version fields for `maturin` and `screed` too), but I have a PR ready to unbreak it as soon as they are updated in `nixpkgs` (both have been merged there already)

Reply:
fixed in https://github.com/sourmash-bio/sourmash/pull/2481